src/node/index.ts

```diff
@@ -70,19 +75,21 @@ class BeaconNode {
       db: this.db,
       chain: this.chain,
     });
-    this.rpc = new JSONRPC(this.conf.rpc, {
+    this.rpc = this.setupRPC();
```

Comment:
I think this setup should happen outside of this node module, probably in the cli setup, and the `BeaconNode` should expect `IApi`s.
Eg:
```typescript
interface RpcCtx {
  apis: IApi[];
}
```
This way we can handle all string/cli parsing in one place. And the BeaconNode can be created more independently of the concerns of the cli.

Reply:
Yeah that makes sense